python/py-us/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        unitedstates python-us 2.0.2
+revision            0
+name                py-us
+
+platforms           darwin
+license             BSD
+maintainers         nomaintainer
+
+description         US: The Greatest Package in the World
+long_description    A package for easily working with US and state metadata.
+
+homepage            https://github.com/unitedstates/python-us
+
+checksums           rmd160  1b00f8384d407659b219173d007d268b43bad365 \
+                    sha256  82f13a0ec452c7870335d44c13214ba29caed9c005bea97138dbce5deb04e2d9 \
+                    size    12315
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
```

Comment:
the port installs a `console_script`, so likely `setuptools` is needed at runtime as well - please check

Reply:
Added thanks.